wowchemy/layouts/partials/navbar.html

```diff
@@ -139,6 +139,37 @@
     </div><!-- /.navbar-collapse -->
 
     <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
+      {{/* Look for social icons that should be displayed in the navbar.
+           For now just look for a superuser. If more than one superuser
+           is defined, run through all of them. */}}
+
+      {{ range where site.Pages "Section" "authors" }}
+        {{ if .Params.superuser }}
```

Comment:
This condition can be added to the query for efficiency

Reply:
Fixed